CefSharp/DependencyChecker.cs

```diff
@@ -137,8 +153,19 @@ public static List<string> CheckDependencies(bool checkOptional, bool packLoadin
                 }
             }
 
+            var browserSubprocessPath = string.IsNullOrEmpty(cefSettings.BrowserSubprocessPath)
```

Comment:
By default `BrowserSubprocessPath` defaults to `CefSharp.BrowserSubprocess.exe` so I think it's probably better just to check the property, if empty throw a unique exception, possibly in `Cef.Initialize()` it's self?